bin/webrtc-server/README.md

```diff
@@ -0,0 +1,51 @@
+# WebRTC server
+
+## Install
+
+The certificate in `./static` directory **MUST** be marked as trusted (Chrome:
+'Security and privacy' -> 'Security' -> 'Manage certificates').
```

Comment:
Why is this necessary?

Reply:
You can discard that. It turned out to be another issue with the library I'm using.